server/llm/providers/google.ts

```
@@ -168,69 +159,63 @@ export async function generateWithGoogle(
-          parameters: tool.parameters,
-        })),
-      },
-    ];
+    const functionDeclarations: FunctionDeclaration[] = params.tools.map(
+      (tool) => ({
+        name: tool.name,
+        description: tool.description,
+        parameters: tool.parameters as Schema,
+      }),
+    );
+    config.tools = [{ functionDeclarations: functionDeclarations }];
```

ESLint

[error] Expected property shorthand.

(object-shorthand)